### DISCUSSION TITLE
Solved: Template problem for AWP12L

### DISCUSSION THREAD

**Discussion by dac2020** [arendst/Tasmota#16306] [Q&A] [answered]
I bought these; https://www.amazon.com/dp/B0828BH1ZK and was able to load the basic tasmota onto them using tuya-convert.
I then used this template; https://templates.blakadder.com/avatar_AWP12L.html to update the device then upgraded.
However, I still see only one plug to control, not two. The template seems to control one single plug on each of the devices.

Am I missing something?

My mistake. I forgot to 'activate' the template after making the change. All is good now. Wish I could delete this.

**Comment by barbudor** [comment, marked as answer]
The template has 2 relays so it should have 2 power buttons in the gui
You may have forgot to activate the template by either checking the 'activate'  box in the configure - other page or by typing command 'module 0' in the console

**Reply by barbudor** [reply]
Oups
Didn't saw your edit 😂
